sdk/communication/azure-communication-chat/README.md

```diff
@@ -69,24 +69,18 @@ The following sections provide several code snippets covering some of the most c
 
 ## Create the chat client
 
-<!-- embedme ./src/samples/java/com/azure/communication/chat/ReadmeSamples.java#L41-L57 -->
+<!-- embedme ./src/samples/java/com/azure/communication/chat/ReadmeSamples.java#L38-L48 -->
 ```Java
 String endpoint = "https://<RESOURCE_NAME>.communcationservices.azure.com";
 
-// Create an HttpClient builder of your choice and customize it
-// Use com.azure.core.http.netty.NettyAsyncHttpClientBuilder if that suits your needs
-NettyAsyncHttpClientBuilder httpClientBuilder = new NettyAsyncHttpClientBuilder();
-HttpClient httpClient = httpClientBuilder.build();
-
```

Comment:
The sample would not work if we remove this since the builder will throw if httpClient is not passed in. It'll be better we say this is just an example and Any implementation of HttpClient will work but it cannot be null

Reply:
This seems to have been taken care now, the httpClient requirement has been removed from the builder